components/centraldashboard/app/api_workgroup.ts

```diff
@@ -27,6 +27,7 @@ interface HasWorkgroupResponse {
     user: string;
     hasAuth: boolean;
     hasWorkgroup: boolean;
+    namespaceCreation: boolean;
```

Comment:
Does this come from [params.env](https://github.com/kubeflow/manifests/blob/68c90fd6212a3044bda1f230dea84df5d299f68f/common/centraldashboard/base/params.env#L1) file as an env variable? Do you need to submit a change for that as well?

Reply:
Right, I can submit a PR on the manifests once this is approved. I think that's lower priority, since you can modify it when you build the manifests.

Reply:
Would it make more sense to perhaps use `disableRegistrationFlow`?
And make the logic such that:
- If this arg is passed then conditionally in the UI allow the registration flow to not execute

Reply:
@avdaredevil I set it to registrationFlow and the logic remains as what it was before the PR, with exception that this variable will dictate if the registration flow is enabled or not. 

Should be good now, there are already a few test cases that ensure the logic is correct.